volsupervisor/snapshots.go

```diff
@@ -11,7 +11,7 @@ import (
 	log "github.com/Sirupsen/logrus"
 )
 
-func wrapSnapshotAction(action func(config *config.TopLevelConfig, pool string, volume *config.VolumeConfig)) func(*volumeDispatch) {
+func wrapSnapshotAction(timeout time.Duration, action func(config *config.TopLevelConfig, pool string, volume *config.VolumeConfig, timeout time.Duration)) func(*volumeDispatch) {
```

Comment:
This is again minor and similar to previous comment may be we should make `wrapSnapshotAction` et-al `Daemon` methods to avoid having to pass around timeout as argument.

I realize supervisor doesn't yet have a config of it's own but may be we can introduce one?